Jenkinsfile.broker

```diff
@@ -45,7 +45,7 @@ pipeline {
                         jobs = ['IntegrationTests']
                     } else {
                         jobs = ['EssCommon', 'ATAOS', 'ATDome', 'ATDomeTrajectory', 'ATHexapod', 'ATMCSsimulator', 'ATPneumaticssimulator', 'ATSpectrograph', 'ATWhiteLightSource', 'Authorize',
-                            'CBP', 'DIMM', 'DREAM', 'DSM', 'EAS', 'Electrometer', 'ExternalScripts', 'FiberSpectrograph', 'GenericCamera', 'hexrotcomm', 'IntegrationTests',
+                            'CBP', 'DIMM', 'DREAM', 'DSM', 'EAS', 'Electrometer', 'ExternalScripts', 'FiberSpectrograph', 'GenericCamera', 'hexrotcomm', 'IntegrationTests', 'LaserTracker',
```

Comment:
Is LaserTracker right? The package name is `ts_lasertracker` (all lowercase).

Also (pre-existing) ATWhiteLightSource is an odd name for a package now named `ts_atwhitelight` (I renamed the package to remove "Source" from the name, in order to match the SAL component name, and make it lowercase to match our convention of lowercase package names).

Reply:
This is name of the Jenkins job, not the conda package or repository name; <CSC>_conda_package.  I just renamed the MTAlignment_conda_package job to LaserTracker_conda_package and pointed it to the renamed repository.